src/VirtoCommerce.NotificationsModule.Web/Scripts/blades/notifications-journal.js

```diff
@@ -67,6 +67,23 @@ angular.module('virtoCommerce.notificationsModule')
                 name: "platform.commands.refresh", icon: 'fa fa-refresh',
                 executeMethod: blade.refresh,
                 canExecuteMethod: function () { return true; }
+            },
+            {
+                name: 'notifications.blades.notification-journal.labels.resend-messages',
+                icon: 'fa fa-repeat',
+                executeMethod: function (blade) {
```

Comment:
The blade is already declared.

```suggestion
                executeMethod: function () {
```